refinery/core/models.py

```diff
@@ -1875,7 +1875,7 @@ def get_managed_group(self):
     def save(self, *args, **kwargs):
```

Comment:
Also, the group name cannot be blank already: name = models.CharField(_('name'), max_length=80, unique=True), so no additional checking for that is necessary. It might be a good idea to strip whitespace characters though.